MetaMask/metamask-extension
# Adds e2e json rpc method tests.

## **Description**
_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:_
_1. What is the reason for the change?_
_2. What is the improvement/solution?_

Adds e2e json rpc method tests for `eth_newBlockFilter`, `eth_accounts`, `eth_gasPrice`, and `eth_estimateGas`.

## **Manual testing steps**

_1. Step1:_
_2. Step2:_
_3. ..._

1. Pull down branch
2. `yarn && yarn build:test`
3. `yarn test:e2e:chrome:rpc`
4. Tests should not fail.

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

_[screenshot]_

### **After**

_[screenshot]_

## **Related issues**

_Fixes #???_

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
